src/sass/components/ATATButton.scss

```diff
@@ -68,6 +68,11 @@
   &.link-button, &.tertirary {
     padding: 6px 12px !important;
 
+    &.inline-text{
+      padding: 0px 4px !important;
```

Comment:
class won't be necessary if changing `v-btn` to `a` tag

Reply:
-[x] removed